rbac/providers/common/admin_setup.py

```diff
@@ -54,13 +95,26 @@ def add_admin_accounts():
         role_response = session.post(
             "http://rbac-server:8000/api/roles", json=admin_role
         )
+        role_response_json = role_response.json()
         if role_response.status_code >= 300:
```

Comment:
If we are checking for the exact 200 response for user creation, we should do the same here when the role is created for consistency.